tests/wpt/metadata/html/semantics/embedded-content/media-elements/event_volumechange.html.ini

```diff
@@ -1,20 +1,20 @@
 [event_volumechange.html]
-  expected: TIMEOUT
+  expected: PASS
```

Comment:
How did you end up with a `PASS` here? I believe it should be `OK`. Did you manually change this file :) ? If that's the case, please, use `./mach update-wpt <log-of-a-mach-wpt-run>` instead as described [here](https://github.com/servo/servo/tree/master/tests/wpt#updating-test-expectations)

Reply:
Yes, I did change that file manually. I will update the expectations using `mach` like you suggest instead